### DISCUSSION TITLE
Cant get API to create Words

### DISCUSSION THREAD

**Discussion by GoudaCouda** [erew123/alltalk_tts#446] [General]
I do not understand I am trying to make api calls either through the UI or through an alternate app but every time it just sounds like glitched out audio. Im pretty sure it is something I am doing wrong as I can get it working with the regular all talk UI but the API just sounds like nothing almost. I dont know how to describe the sound it is just like a buzzing noise of the tts model trying to make out sounds.

I am using XTTS and am trying streaming and regular.

**Comment by erew123** [comment]
Hi @GoudaCouda here is a full JavaScript example of a streaming call and using a browser audio element to play back. https://github.com/erew123/alltalk_tts/wiki/API-%E2%80%90-Streaming-TTS-Generation-API#javascript-example-for-streaming-playback

You can also find examples within admin.html

https://github.com/erew123/alltalk_tts/blob/alltalkbeta/system/admin.html#L803

You've not specified exactly what it is you are playing back audio streaming in? Is it a web browser or something like unity or something else?
